docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -23,8 +23,8 @@ experimental[]
 To visualize data in a CSV, JSON, or log file, you can upload it using the File 
 Data Visualizer. On the home page, click *Import a CSV, NDSON, or log file*, and 
 then drag your file into the File Data Visualizer. Alternatively, you can open 
-it by navigating to the Machine Learning app page from the sidebar menu and 
-selecting the Data Visualizer from the top navigation bar on the opening page.
+it by navigating to Machine Learning from the side navigation and selecting 
+Data Visualizer.
```

Comment:
Likewise, the guidance is to "Bold the names of elements that you’re directing the user to interact with"
```suggestion
*Data Visualizer*.
```